api/firewaller/application.go

```diff
@@ -71,6 +56,9 @@ func (s *Application) IsExposed() (bool, error) {
 	}
 	result := results.Results[0]
 	if result.Error != nil {
+		if params.IsCodeNotFound(result.Error) {
+			return false, errors.NewNotFound(result.Error, "")
```

Comment:
Would it not be wise to add some message instead of `""`? (nit at best!)

Reply:
You still get the same error message as on the server, complete with the "not found" suffix. This just eschews further message decoration.